### PR TITLE
Federated bridge pallet

### PR DESCRIPTION
**Pull request by kacperzuk-neti**

### CODE REVIEW

File: frame/llm/src/impl_fungible.rs [hunk 1]
```diff
@@ -0,0 +1,71 @@
+use super::*;
+use frame_support::{
```
[the remark this answers is not in the document]
Comment (kacperzuk-neti) [reply frame/llm/src/impl_fungible.rs:2]: Bridge can bridge any asset that implements standard substrate `Transfer` trait. `pallet_balances` (LLD) has it out of the box, for LLM we need to implement it, which is done in this file.

File: frame/federated-bridge/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,837 @@
...
+	pub(super) type Fee<T: Config<I>, I: 'static = ()> =
+		StorageValue<_, BalanceOf<T, I>, ValueQuery>;
+
+	#[pallet::storage]
+	/// Bridge state
+	pub(super) type State<T: Config<I>, I: 'static = ()> = StorageValue<_, BridgeState, ValueQuery>;
+
+	#[pallet::storage]
+	/// Receipts - details on eth -> substrate transfers
+	pub(super) type Receipts<T: Config<I>, I: 'static = ()> = StorageMap<
```
[the remark this answers is not in the document]
Comment (DorianSternVukotic) [reply frame/federated-bridge/src/lib.rs:340]: Yes

File: frame/llm/src/impl_fungible.rs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+		dest: &T::AccountId,
+		amount: Self::Balance,
+		_keep_alive: bool,
+	) -> Result<Self::Balance, DispatchError> {
+		Pallet::<T>::transfer(source.clone(), dest.clone(), amount)?;
+		Ok(amount)
+	}
+
+	fn deactivate(_: Self::Balance) {
+		unimplemented!();
```
Comment (DorianSternVukotic) [inline frame/llm/src/impl_fungible.rs:65]: What does this need to be?

Comment (kacperzuk-neti) [reply frame/llm/src/impl_fungible.rs:65]: https://github.com/paritytech/substrate/blob/polkadot-v0.9.37/frame/support/src/traits/tokens/fungible.rs#L130-L134

This is a global lock for some part of total token issuance. We don't support this for LLM in any way, hence "unimplemented".
It doesn't matter, as Bridge doesn't use these 2 methods.

File: frame/federated-bridge/src/lib.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (kacperzuk-neti) [reply frame/federated-bridge/src/lib.rs]: Done

File: bin/node/runtime/src/lib.rs [hunk 4]
```diff
@@ -1521,6 +1520,64 @@ impl pallet_office::Config<AssetRegistryOfficeInstance> for Runtime {
...
+	pub const WithdrawalDelay: BlockNumber = 60 * MINUTES;
+
+	// * 66k LLD in single hour (1 burst + hour avg)
+	// * 894k LLD in single day (1 burst + 24*hour avg)
+	pub const LLDRateLimit: (Balance, Balance) = (
+		// max burst and max single withdrawal
+		30_000 * DOLLARS,
+		// decay per block (max average withdrawal rate over infinite time)
+		// 60 LLD * 600 blocks/h = max avg 36k LLD per hour
+		60 * DOLLARS
```
Comment (DorianSternVukotic) [inline bin/node/runtime/src/lib.rs:1537]: likely less, as 1.3 mil is the total issuance, but ok, we can change it for mainnet

**Pull request merged**